internal/statemachine/classic_states.go

```diff
@@ -315,9 +333,19 @@ func (stateMachine *StateMachine) germinate() error {
 	germinateCmd := generateGerminateCmd(classicStateMachine.ImageDef)
 	germinateCmd.Dir = germinateDir
 
-	if germinateOutput, err := germinateCmd.CombinedOutput(); err != nil {
+	var germinateOutput bytes.Buffer
+	germinateCmd.Stdout = &germinateOutput
+	germinateCmd.Stderr = &germinateOutput
+	if classicStateMachine.commonFlags.SubCmdOutput {
+		mwriter := io.MultiWriter(os.Stdout, &germinateOutput)
+		germinateCmd.Stdout = mwriter
+		germinateCmd.Stderr = mwriter
+	}
```

Comment:
Repeating this code each time doesn't feel scalable, some kind of utility surely?